qiskit_nature/operators/second_quantization/second_quantized_op.py

```diff
@@ -48,7 +49,7 @@ def simplify(self, atol: Optional[float] = None):
         Returns a new operator (the original operator is not modified).
 
         Args:
-            atol: Absolute tolerance for checking if coefficients are zero (Default: 1e-8).
+            atol: Absolute numerical tolerance. The default behavior is to use ``self.atol``.
```

Comment:
In Terra's docs, the default value of `atol` can be inferred by Sphinx from the `TolerancesMeta` class, but this does not work within the separate docs built for Nature. Thus, I would like to retain the default value in the docstring here. I.e. say that the default is to use `self.atol`, which in turn defaults to `1e-8`

Reply:
Done.